packages/app/obojobo-express/server/routes/api/visits.js

```diff
@@ -111,7 +139,10 @@ router
 					isPreviewing: req.currentVisit.is_preview,
 					lti,
 					viewState,
-					extensions: visitStartExtensions
+					extensions: visitStartExtensions,
+					isNotificationEnabled,
+					notificationTitle,
+					notificationText
```

Comment:
See above comment: rather than informing the frontend of a single notification with three new properties, try to provide all valid notifications as objects in a single array.